### DISCUSSION TITLE
Error Control Net forge ui torch.Size

### DISCUSSION THREAD

**Discussion by mrsunshine74** [lllyasviel/stable-diffusion-webui-forge#556] [Q&A]
Hi, I find forge great but I have a problem with control net. Everything works fine at the generation level, but as soon as I insert a control net (dept/canny) I encounter this problem::
warning control could not be applied torch.Size([1, 1280, 16, 16]) torch.Size([1, 1280, 32, 32])
warning control could not be applied torch.Size([1, 640, 16, 16]) torch.Size([1, 640, 32, 32]) 
and the generation does not respect control net even though the maps are displayed correctly at the end of the generation! What can I do to fix it? Thanks in advance for the help!

**Comment by auraeinteractive** [comment]
This issue happens also in combination with Kohya HrFix.
